gstreamer/media_manager.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+from gi.repository import Gst as gst
+from gi.repository import GLib
```

Comment:
Alphabetical order for includes `glib` before `gst`

Reply:
The formatter tool order it in that order due to the upper case "L".